app/models/tslr_claim.rb

```diff
@@ -84,7 +84,7 @@ class TslrClaim < ApplicationRecord
   validate  :bank_account_number_must_be_eight_digits
   validate  :bank_sort_code_must_be_six_digits
 
-  before_validation :update_current_school, if: :employment_status_changed?
+  before_validation :reset_inferred_current_school, if: ->(record) { record.persisted? && record.employment_status_changed? }
```

Comment:
Cool, I didn't know about `persisted?`. That's a nice solution 👍

Reply:
yep, it's the opposite of `new_record?`